interfaces/Portalicious/src/app/pages/user-settings/user-settings.component.html

```diff
@@ -1,3 +1,131 @@
+              <app-form-error
+                [visible]="true"
+                error="This field is required."
+                i18n-error="@@generic-required-field"
+              />
+            }
+            @if (changePasswordForm.controls.newPassword.errors?.minlength) {
+              <app-form-error
+                [visible]="true"
+                error="The new password must be at least 8 characters long."
+                i18n-error
+              />
+            }
+            @if (
+              changePasswordForm.controls.newPassword.errors?.newEqualsCurrent
+            ) {
+              <app-form-error
+                [visible]="true"
+                error="The new password must be different from the current password."
+                i18n-error
+              />
+            }
+            <!-- eslint-disable-next-line @angular-eslint/template/label-has-associated-control -->
+            <label>
+              <ng-container i18n>Confirm Password</ng-container>
+              <p-password
+                formControlName="confirmPassword"
+                [toggleMask]="true"
+                [feedback]="false"
+                styleClass="w-full [&_input]:w-full mt-2"
+              />
+            </label>
+            @if (changePasswordForm.controls.confirmPassword.errors?.required) {
+              <app-form-error
+                [visible]="true"
+                error="This field is required."
+                i18n-error="@@generic-required-field"
+              />
+            }
+            @if (
+              changePasswordForm.controls.confirmPassword.errors
+                ?.confirmDifferentFromNew
+            ) {
+              <app-form-error
+                [visible]="true"
+                error="The confirm password must be equal to the new password."
+                i18n-error
+              />
+            }
+            <p-button
+              label="Change password"
+              i18n-label
+              type="submit"
+              rounded
+              class="mt-8"
+              [disabled]="!changePasswordForm.valid"
+              i18n-disabled
+            />
+          </div>
+        } @else if (changePasswordMutation.isPending()) {
+          <p-progressSpinner
+            aria-label="Loading"
+            i18n-aria-label="@@generic-loading"
+          />
+        } @else {
+          @if (changePasswordError()) {
+            <p-messages [severity]="'error'">
+              <ng-template pTemplate>
+                <i class="pi pi-check-circle"></i>
+                <div class="ml-2">
+                  {{ changePasswordError() }}
+                </div>
+              </ng-template>
+            </p-messages>
+          } @else {
+            <p-messages [severity]="'success'">
+              <ng-template pTemplate>
+                <i class="pi pi-exclamation-triangle"></i>
+                <div
+                  i18n
+                  class="ml-2"
```

Comment:
We should not use "margin-**left**", as we want to have the interface compatible with RTL-languages..

We should use "margin-inline-**start**", so `ms-2`.

```suggestion
                  class="ms-2"
```